cgmanifest.json

```diff
@@ -28967,8 +28967,8 @@
         "type": "other",
         "other": {
           "name": "vala",
-          "version": "0.54.6",
-          "downloadUrl": "http://download.gnome.org/sources/vala/0.54/vala-0.54.6.tar.xz"
+          "version": "0.54.14",
```

Comment:
should be `0.56.14` and `https://download.gnome.org/sources/vala/0.56/vala-0.56.14.tar.xz`